gradle-compat-6-1/src/main/java/wtf/emulator/GradleCompat_6_1.java

```diff
@@ -3,13 +3,25 @@
 import org.gradle.api.Project;
 import org.gradle.api.provider.Provider;
 
+import javax.annotation.Nullable;
+
 public class GradleCompat_6_1 implements GradleCompat {
   @Override
   public boolean isConfigurationCacheEnabled() {
     // config cache starting from 6.6
     return false;
   }
 
+  @Nullable
+  @Override
+  public String getGradleProperty(Project project, String name) {
+    Object value = project.getRootProject().findProperty(name);
```

Comment:
We use `getRootProject` here to match the semantics of `providers.gradleProperty` which only look in the root project